### PR TITLE
rename model names

### PR DESCRIPTION
**Pull request by alanhc**
修復檔案命名問題

### CODE REVIEW

File: controllers/courses_controller.js [hunk 1]
```diff
@@ -1,15 +1,15 @@
-const getCourses = require("../models/courseDetail_model");
-const getComment = require("../models/getComment_model");
-const updateComment = require("../models/updateComment_model");
-const deleteComment = require("../models/deleteComment_model");
-const getCourseList = require("../models/courseList_model");
-const getCourse = require("../models/getCourses_model");
-const saveComment = require("../models/comment_model");
+const getCourses = require("../models/getRating_model");
+const getComment = require("../models/_getComment_model");
+const updateComment = require("../models/_updateComment_model");
```
Comment (JianShan-1214) [inline controllers/courses_controller.js:3]: done

File: controllers/courses_controller.js [hunk 2]
```diff
@@ -1,15 +1,15 @@
...
+const deleteComment = require("../models/_deleteComment_model");
```
Comment (JianShan-1214) [inline controllers/courses_controller.js:4]: done

File: controllers/courses_controller.js (hunk 1 above)
Comment (JianShan-1214) [inline controllers/courses_controller.js:3]: done

File: controllers/courses_controller.js [hunk 3]
```diff
@@ -1,15 +1,15 @@
...
+const getCourseList = require("../models/_courseList_model");
+const getCourse = require("../models/_getCourses_model");
+const saveComment = require("../models/_comment_model");
 const saveLog = require("../models/saveError_model");
 const getLog = require("../models/getLog_model");
 const getUser = require("../models/getUser_model");
 const countDB = require("../models/dbCount_model");
-const checkComment = require("../models/checkComment_model");
+const checkComment = require("../models/_checkComment_model");
```
Comment (JianShan-1214) [inline controllers/courses_controller.js:12]: done

**Review by fanlan1210 — COMMENTED** [review]
it seems OK. Waiting for @JianShan-1214 's approvement.

File: controllers/courses_controller.js [hunk 4]
```diff
@@ -1,15 +1,15 @@
-const getCourses = require("../models/courseDetail_model");
-const getComment = require("../models/getComment_model");
-const updateComment = require("../models/updateComment_model");
-const deleteComment = require("../models/deleteComment_model");
-const getCourseList = require("../models/courseList_model");
-const getCourse = require("../models/getCourses_model");
-const saveComment = require("../models/comment_model");
+const getCourses = require("../models/getRating_model");
```
Comment (JianShan-1214) [inline controllers/courses_controller.js:1]: getCourses本來說要合併，但後來覺得應該是兩段不同的東西，所以我覺得應該不用合併

File: controllers/courses_controller.js [hunk 5]
```diff
@@ -1,15 +1,15 @@
...
+const getComment = require("../models/_getComment_model");
+const updateComment = require("../models/_updateComment_model");
+const deleteComment = require("../models/_deleteComment_model");
+const getCourseList = require("../models/_courseList_model");
+const getCourse = require("../models/_getCourses_model");
```
Comment (JianShan-1214) [inline controllers/courses_controller.js:6]: 同上

**Comment by fanlan1210** [comment]
conflict has been fixed in #7

**Pull request closed without merging**